cmd/transferer/datasources.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/valyala/fasthttp"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type dataSourceResp struct {
+	Id          int         `json:"id"`
+	OrgId       int         `json:"orgId"`
+	Name        string      `json:"name"`
+	Type        string      `json:"type"`
+	TypeLogoUrl string      `json:"typeLogoUrl"`
+	Access      string      `json:"access"`
+	URL         string      `json:"url"`
+	Password    string      `json:"password"`
+	User        string      `json:"user"`
+	Database    string      `json:"database"`
+	BasicAuth   bool        `json:"basicAuth"`
+	IsDefault   bool        `json:"isDefault"`
+	JsonData    interface{} `json:"jsonData"`
+	ReadOnly    bool        `json:"readOnly"`
+}
+
+type DataSources struct {
```

Comment:
rename to PMMConfig and fields

Reply:
it should contain:
PMM URL
CH URL
VM URL

Reply:
Resolved in 8ddcf0da92e6d41ed9759a2e07e04f06f1581e4f